.github/workflows/ci.yml

```diff
@@ -31,20 +31,18 @@ jobs:
 
       - name: Compute node modules cache key
         id: nodeModulesCacheKey
-        run: echo "{value}=$(node build/azure-pipelines/common/computeNodeModulesCacheKey.js)" >> $GITHUB_OUTPUT
-      - name: Cache node_modules archive
+        run: echo "value=$(node build/azure-pipelines/common/sql-computeNodeModulesCacheKey.js)" >> $ENV:GITHUB_OUTPUT
```

Comment:
Is the ENV added here because it's running on Windows so defaults to powershell? (vs the basic workflow on Ubuntu)

Reply:
Yes, on windows env variables are handled as `$env:`